src/Components/benchmarkapps/Wasm.Performance/Driver/Program.cs

```diff
@@ -100,17 +102,11 @@ private static void FormatAsBenchmarksOutput(List<BenchmarkResult> results)
                 Format = "n2",
             });
 
-            var testAssembly = typeof(TestApp.Startup).Assembly;
-            var testAssemblyLocation = new FileInfo(testAssembly.Location);
-            var testApp = new DirectoryInfo(Path.Combine(
-                testAssemblyLocation.Directory.FullName,
-                testAssembly.GetName().Name));
-
             output.Measurements.Add(new BenchmarkMeasurement
             {
                 Timestamp = DateTime.UtcNow,
-                Name = "Publish size",
-                Value = GetDirectorySize(testApp) / 1024,
+                Name = "blazorwasm/publish-size",
```

Comment:
No metadata? It's not mandatory, just wondering.

Reply:
It's over here; https://github.com/dotnet/aspnetcore/pull/18485/files/ef69bb5f8ea26a052a5a7320cf86d924d1d5777c#diff-5c91afacb27c39e7f1cdb4f2ee95a0d5R99. Hadn't changed in the diff.